.github/archive/test_failures.yml

```diff
@@ -0,0 +1,12 @@
+name: OnPurpose Failure
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+jobs:
+  failure_test:
+    if: contains(github.repository, 'dougch/s2n')
+    runs-on: ubuntu-latest
+    steps:
+      - name: give the failure check something to look at
+        run: exit 1
```

Comment:
/nit new line

Reply:
file removed- it was for my private testing only.